system_up.sh

```diff
@@ -20,6 +20,8 @@ start_services() {
 		done
 	fi
 	echo "👷 Building service(s)"
+
+  cp ../Dockers/config/litefs.static-lease.yml ./config/
```

Comment:
Can you use `cp "$(realpath $(dirname $0))/Dockers/config/litefs.static-lease.yml" "$(realpath $(dirname $0))/contracts/config/"` so we can launch the script from any directory instead pls